bpf/kmesh/ads/include/cluster.h

```diff
@@ -219,14 +247,95 @@ static inline void *loadbalance_round_robin(struct cluster_endpoints *eps)
     return (void *)eps->ep_identity[idx];
 }
 
-static inline void *cluster_get_ep_identity_by_lb_policy(struct cluster_endpoints *eps, __u32 lb_policy)
+/* The daddr is explicitly excluded from the hash here in order to allow for
+ * backend selection to choose the same backend even on different service VIPs.
+ */
+// static __always_inline __u32 hash_from_tuple_v4(struct bpf_sock * sk)
+// {
+
+// 	BPF_LOG(INFO, CLUSTER, "sk: src_ip is:%x, src_port is:%d\n", sk->src_ip4,sk->src_port);
+// 	BPF_LOG(INFO, CLUSTER, "sk: dst_port is:%d, sk->protocol is: %x\n",sk->dst_port,sk->protocol);
+// 	return jhash_3words(sk->src_ip4,
+// 			    ((__u32) sk->dst_port << 16) | sk->src_port,
+// 			    sk->protocol, HASH_INIT4_SEED);
+// }
+
+static __always_inline __u32 map_array_get_32(__u32 *array, __u32 index, const __u32 limit)
```

Comment:
This function is used to obtain a u32 data from an array. The function name can be optimized.

Reply:
Will fix in next Pr.